container-aws-python/__main__.py

```diff
@@ -0,0 +1,41 @@
+from pulumi import Config, Output, export
+import pulumi_aws as aws
+import pulumi_awsx as awsx
+
+config = Config()
+container_port = config.get_int("containerPort") or 80
+cpu = config.get_int("cpu") or 512
+memory = config.get_int("memory") or 128
+
+# An ECS cluster to deploy into
+cluster = aws.ecs.Cluster("cluster")
+
+# An ALB to serve the container endpoint to the internet
+loadbalancer = awsx.lb.ApplicationLoadBalancer("loadbalancer")
+
+# An ECR repository to store our application's container image
+repo = awsx.ecr.Repository("repo")
+
+# Build and publish our application's container image from ./app to the ECR repository
+image = awsx.ecr.Image("image",
```

Comment:
General PEP8 nit: There will be warnings on opening this in an editor with checks turned on as indentation for lines 21-22 and 26-38 are not visually aligned to the first argument. Two options for fixing this one:
* Option 1: Move the first argument to a new line:
```python
image = awsx.ecr.Image(
    "image",
    repository_url=repo.url,
    path="./app"
)
```
* Option 2: Move everything over to visually indent:
```python
image = awsx.ecr.Image("image",
                       repository_url=repo.url,
                       path="./app")
```
Both options are equally valid and idiomatic. I prefer the first as I'm less likely to hit line length linters and I find it more readable as things get more complicated, but that's a personal preference.